js/main.js

```diff
@@ -0,0 +1,91 @@
+var button = document.getElementById('buttonNews');
+button.onclick = function displayAndHide() {
+    var hide = document.getElementById('displayInfo');
+    if (hide.style.display === "") {
+        hide.style.display = 'none';
+    } else {
+        hide.style.display = '';
+    }
+}
+
+var submitBtn = document.querySelector('#submitBtn');
+submitBtn.addEventListener("click", checkName);
+
+function checkName(evt) {
+    evt.preventDefault();
+    var nameArea = document.querySelector("#nameArea");
+    if (nameArea.value == "") {
+        alert("Error: Name is empty!");
+        nameArea.focus();
+        return false;
+    }
+}
+
+submitBtn.addEventListener("click", checkEmail);
+
+function checkEmail(evt) {
+    evt.preventDefault();
+    var emailArea = document.querySelector("#emailArea");
+    var filter = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+    if (emailArea.value == "") {
+        alert("Error: Email is empty!");
+        emailArea.focus();
+        return false;
+    }
+    if (!filter.test(emailArea.value)) {
+        alert('Please provide a valid email address');
+        emailArea.focus;
+        return false;
+    }
+}
+
+submitBtn.addEventListener("click", checkMobileNumber);
+function checkMobileNumber(number) {
+    number.preventDefault();
```

Comment:
it is not **number**, it is event (if it is number, then you shouldn't call .preventDefault)